examples/collect_links/collect_links.rs

```diff
@@ -21,7 +21,7 @@ async fn main() -> Result<()> {
         },
     ];
 
-    let links = Collector::new(None) // base
+    let links = Collector::new(None, None) // root_path and base
```

Comment:
I wonder if we should use `Collector::default` here, where `base` and `root_path` are set to `None`.

Reply:
sure, done